script.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+let total = 0;
+let highest = 0;
+let stuAvg= 0;  
+var names= [];
+var grade= [];
+var pos1 = [];
+var arr = [];
+
+for (var i = 0; i < 10; i++) {             
+    names.push(prompt('Enter The Ten Student Names | Student: ' + (i + 1))); 
+    document.getElementById("stuNames").innerHTML=("Student's Names:" +names.join(','));
+          
+    grade.push(Number(prompt('Enter The Ten Student Grades | Grade: ' + (i + 1))));
+    total += grade[i] 
+      
+    stuAvg = total / grade.length;
+    highest = Math.max(...grade);
+    pos1 = grade.indexOf(highest);
+    document.getElementById("stuGrades").innerHTML=("Student's Grades:" +grade.join(','));
+}
+
+
+function results() {
+    document.getElementById("result").innerHTML=("Highest Grade:"+highest+"The Highest Grade Belongs to:"+names[pos1]+" The Average Grade is "+stuAvg+"");
+}  
+
+function passingStu() {
+
+    grade.filter(passing); 

```

Comment:
why is the passing helper function defined after it's being passed to the array filter? Also you could just have the function be defined in the filter itself ```grade.filter(grade => grade > 60)```